App/components/Bible/Chapter.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
```

Comment:
This theme setup will have to happen for all components that are wrapped with the ThemeProvider, so it might be good to figure out a global setup for tests so this doesn't have to be done every time.